src/tests/admin/conferenceRoom/ForARoomDisabledTheTabletDoesNotScheduleMeetings.java

```diff
@@ -0,0 +1,58 @@
+package tests.admin.conferenceRoom;
+
+import static framework.common.AppConfigConstants.EXCEL_INPUT_DATA;
+
+import java.util.List;
+import java.util.Map;
+
+import org.testng.Assert;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import framework.pages.admin.HomeAdminPage;
+import framework.pages.admin.conferencerooms.RoomsPage;
+import framework.pages.tablet.HomeTabletPage;
+import framework.pages.tablet.SearchPage;
+import framework.utils.readers.ExcelReader;
+
+/**
+ * TC24: Verify that when a room status is disabled, the tablet does not schedule meetings
+ * @author Juan Carlos Guevara
+ */
+public class ForARoomDisabledTheTabletDoesNotScheduleMeetings {
+	
+	//Reading resource data from an .xls file
+	private ExcelReader excelReader = new ExcelReader(EXCEL_INPUT_DATA);
+	private List<Map<String, String>> testData = excelReader.getMapValues("Resources");
+	private String roomName = testData.get(1).get("Room Name");
+
+	@BeforeClass(groups = "FUNCTIONAL")
+	public void disablingARoom(){
+
+		//Disable room in Admin 
+		HomeAdminPage homeAdminPage = new HomeAdminPage();
+		RoomsPage roomsPage = homeAdminPage.clickConferenceRoomsLink();		
+		roomsPage.enableDisableIcon(roomName);		
+	}
+
+	@Test(groups = "FUNCTIONAL")
+	public void testForARoomDisabledTheTabletDoesNotScheduleMeetings() {
+
+		//Open tablet to see changes
+		HomeTabletPage homeTabletPage = new HomeTabletPage();
+		SearchPage searchPage = homeTabletPage.clickSearchBtn()
+				.clickCollapseAdvancedBtn()
+				.setName(roomName);
+
+		//Assertion for TC24 
+		Assert.assertFalse(searchPage.roomIsDiplayed(roomName));		
+	}
+
+	@AfterClass(groups = {"FUNCTIONAL"})
+	public void EnablingTheRoom() {
+		HomeAdminPage homeAdminPage = new HomeAdminPage();
+		RoomsPage roomsPage = homeAdminPage.clickConferenceRoomsLink();	
+		roomsPage.enableDisableIcon(roomName);
+	}
```

Comment:
There might be a problem with this test case, if it would be the first to be executed that verifies something in tablet, It doesn't select a room in Tablet's settings, Therefore it would failed, and if you add the steps to select a room, I am not sure if it would select it because the room would be disabled. I suggest to put the steps to select a room at the beforeSuit method